README.md

```diff
@@ -75,6 +75,22 @@ You can also use custom host and port.
 * Activate virtual environment if you wish to do so
 * Navigate into `entrypoint` folder
 * Run `pip install -r requirements.txt`
+* Navigate into `microservice/config` folder
+* Create `.env` file with contents   
+```
+DEVELOPMENT_DATABASE_USER = 
```

Comment:
Should we use the short version of the environments?

`DEVELOPMENT_*` to `DEV_*`
`PRODUCTION_*` to `PROD_*`